src/student.rb

```diff
@@ -2,7 +2,7 @@
 
 class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+    super(name, age, parent_permission: parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] The issue pointed out by the previous reviewer still persists. Kindly inspect it well. The way your constructor here is set up the age and name are interchanged. Based on your Person class, the order of the parameters passed to the Student should be `age`, before `name`

  ```
  super(age, name, parent_permission: parent_permission)
  ```

  ![kip](https://user-images.githubusercontent.com/54848363/195627681-cdee5ff9-7f1a-4e41-83d2-89cb5d466f76.jpg)

  This does not resolve it though. See next suggestion.